homeassistant/components/hassio/websocket_api.py

```diff
@@ -41,9 +41,13 @@
 
 # Endpoints needed for ingress can't require admin because addons can set `panel_admin: false`
 # pylint: disable=implicit-str-concat
+# fmt: off
 WS_NO_ADMIN_ENDPOINTS = re.compile(
-    r"^(?:" r"|/ingress/(session|validate_session)" r"|/addons/[^/]+/info" r")$"
+    r"^(?:"
+    r"|/ingress/(session|validate_session)"
+    r"|/addons/[^/]+/info" r")$" # noqa: ISC001
```

Comment:
```suggestion
    r"^(?:|/ingress/(session|validate_session)|/addons/[^/]+/info)$"
```

Reply:
The suggested pattern has been explicitly avoided in multiple places, including (and not limited by) the Supervisor itself.

I don't want to mess with that.

../Frenck